pkg/appgw/configbuilder_test.go

```diff
@@ -685,8 +685,8 @@ var _ = Describe("Tests `appgw.ConfigBuilder`", func() {
 			},
 			ObjectMeta: metav1.ObjectMeta{
 				Annotations: map[string]string{
-					annotations.IngressClassKey: annotations.ApplicationGatewayIngressClass,
-					annotations.SslRedirectKey:  "true",
+					annotations.IngressClassKey:      annotations.ApplicationGatewayIngressClass,
```

Comment:
Are these tabs ? or extra spaces ?

Reply:
these are spaces! I ran `go fmt ./...` on the entire project.